### PR TITLE
Update blast to new blast (>2.6)

### PR DESCRIPTION
**Pull request by lukasmueller**
Description 
-----------------------------------------------------

Note that this branch requires the Debian blast package ncbi-blast+ to be installed. The legacy interface is not used anymore at all, so both the blast2 and ncbi-blast+-legacy package should not be required.

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [ ] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

File: lib/CXGN/Blast/Parse/Plugin/#Basic.pm# [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (nickmorales) [inline lib/CXGN/Blast/Parse/Plugin/#Basic.pm#]: should this buffer (#Basic.pm#) be added?

Comment (lukasmueller) [reply lib/CXGN/Blast/Parse/Plugin/#Basic.pm#]: no... that should be removed. Good catch. Will do.

Comment (lukasmueller) [reply lib/CXGN/Blast/Parse/Plugin/#Basic.pm#]: Done!

**Comment by lukasmueller** [comment]
I think you may have to re-review....

**Comment by lukasmueller** [comment]
or not

**Pull request merged**